sdcm/nemesis.py

```diff
@@ -1716,6 +1712,33 @@ def disrupt_network_start_stop_interface(self):  # pylint: disable=invalid-name
         finally:
             self.target_node.remoter.run("sudo /sbin/ifup eth1")
 
+    def disrupt_add_del_nodes(self):
```

Comment:
please change name, to "grow_shrink_cluster"

Reply:
fixed